pom.xml

```diff
@@ -1416,58 +1304,84 @@
+                                </includes>
+                                <trimTrailingWhitespace/>
+                                <endWithNewline/>
+                                <licenseHeader>
+                                    <file>license.xml.header</file>
+                                    <!-- Delimiter is a regex which identifies the "top" of the file i.e. all above it is considered a header -->
+                                    <delimiter><![CDATA[(\n)*^(<[^!?]+|<!DOCTYPE.+)$]]></delimiter>
+                                    <skipLinesMatching><![CDATA[<\?xml.+\?>]]></skipLinesMatching>
+                                </licenseHeader>
+                                <indent>
+                                    <tabs>true</tabs>
+                                    <spacesPerTab>4</spacesPerTab>
+                                </indent>
+                            </format>
+                        </formats>
+                        <!-- define a language-specific format -->
+                        <java>
+                            <toggleOffOn/>
+                            <includes>
+                                <include>src/**/*.java</include>
+                            </includes>
+                            <excludes>
+                                <exclude>**/TypeHelper.java</exclude>
+                                <exclude>**/TypeHelperTest.java</exclude>
+                                <exclude>**/ConcurrentReferenceHashMap.java</exclude>
+                            </excludes>
+                            <indent>
+                                <tabs>true</tabs>
+                                <spacesPerTab>4</spacesPerTab>
+                            </indent>
+
+                            <importOrder>
+                                <!-- you can use an empty string for all the imports you didn't specify explicitly, '|' to join group without blank line, and '\#` prefix for static imports. -->
+                                <order>\#,java,javax,jakarta,org.openjdk.jmh,org.hibernate,org.hibernate.testing,org.hibernate.test,org.junit,org.jboss,com,javafx.beans,</order>
+                            </importOrder>
+                            <trimTrailingWhitespace/>
+                            <endWithNewline/>
+                            <eclipse>
+                                <!-- optional version of Eclipse Formatter -->
+                                <!-- <version>4.26</version> -->
+                                <file>hibernate_validator_style.xml</file>
+                            </eclipse>
+                            <removeUnusedImports>
+                                <!-- default one fails to process the classes -->
+                                <engine>cleanthat-javaparser-unnecessaryimport</engine>
+                            </removeUnusedImports>
+                            <licenseHeader>
```

Comment:
Does this apply the header automatically, on top of just checking?

Reply:
it's like with those other plugins, if `spotless:apply` is executed then the headers are added, and if it's `spotless:check` then it fails the build if the header is missing...